### PR TITLE
feat: added login page

### PR DESCRIPTION
**Pull request by kedark152**
### #feat: added login page
1. added email address, password, remember me fields along with forgot password link.
2. added Login and create new account buttons

**Preview Link:** https://deploy-preview-7--agrostores.netlify.app/pages/login.html

### CODE REVIEW

File: css/page-styles/login.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}
+.login-card{
+   display: flex;
+   flex-direction: column;
+   width: 28rem;
+   border-radius: 10px;
+   background-color: var(--white);
+   box-shadow: var(--box-shadow-1);
+   height: fit-content;
+   padding: 1rem 1.5rem ;
```
Comment (AnuragSahu11) [inline css/page-styles/login.css]: You could declare variables for different common space units like 1 rem

**Review by RakshithBhat03 — APPROVED** [review]
The code looks clean and well structured. Great use of variables and comments. Good to merge.

**Review by AnuragSahu11 — APPROVED** [review]
Good work , green color for theme looks good and unique

File: css/page-styles/login.css (hunk 1 above)
Comment (kedark152) [reply css/page-styles/login.css]: Thanks Anurag!! Updated the spacing size with its variables in new commit.

**Pull request merged**